libflux/flux-core/src/semantic/flatbuffers/tests.rs

```diff
@@ -108,6 +116,79 @@ re !~ /foo/
     }
 }
 
+#[test]
+fn test_serialize_vectorization() {
```

Comment:
Can you add a comment here that describes why you needed to add this extra test case? In other words, explain that the normal parse/analyze sequence won't populate the `vectorized` field, so you need to do somethign special here to do that.